File: DayThree/JavaScript/Checkout.js
const prompt = require("prompt-sync")()

    let userItems = []
    let items = []
    let price = []
    let total = []
    
    console.log("=============== WELCOME TO RYAN'S SUPERMARKET ================");

    let index = 0;

    let customer = String(prompt("Enter customer name: "));

    do{
    userItems[index] = String(prompt("Enter product name: "));
    price[index] = Number(prompt("Enter Item Price: "));
    items[index] = Number(prompt("Enter quantity: "));
    total[index] = price[index] * items[index];
    index++;

    let choice = String(prompt("Add more items? (yes/no): ")).toLowerCase();
    if (choice === "no"){
    break;
    }
}
    while (index < 100);

    let totalPrice = 0;
    for(let count = 0; count < index; count++){
    totalPrice += total[count];
}

    let discount = totalPrice * 0.10;
    let vat = totalPrice * 0.075;
    let billTotal = totalPrice - discount + vat;

    console.log("\n===== RECEIPT =====");
    console.log("Customer name: " + customer);

    for (let count = 0; count < index; count++) {
    console.log(userItems[count] + " x" + items[count] + " @ " + price[count] + " = " + total[count]);
}

    console.log("-------------------------");
    console.log(`totalPrice: ${totalPrice.toFixed(2)}\n`);
    console.log(`Discount: ${discount.toFixed(2)}\n`);
    console.log(`VAT (7.5%%): ${vat.toFixed(2)}\n`);
    console.log(`Bill Total: ${billTotal.toFixed(2)}\n`);

    console.log
    let paid = Number(prompt("Amount Paid: "));

    let balance = paid - billTotal;

    console.log(`Balance: ${balance.toFixed(2)}\n`);
    console.log("==================");
    console.log("    THANK YOU");

File: DayThree/Java/Checkout.java
import java.util.Scanner;

public class CheckoutApp{
    public static void main(String... args){

    Scanner input = new Scanner(System.in);

    String[] userItems = new String[100];
    int[] items = new int[100];
    double[] price = new double[100];
    double[] total = new double[100];
    
    System.out.println("=============== WELCOME TO SEMICOLON SUPERMARKET ================");

    int index = 0;

    System.out.print("Enter customer name: ");
    String customer = input.nextLine();

    do{
    System.out.print("Enter product name: ");
    userItems[index] = input.next();

    System.out.print("Enter Item Price: ");
    price[index] = input.nextDouble();

    System.out.print("Enter quantity: ");
    items[index] = input.nextInt();

    total[index] = price[index] * items[index];
    index++;

    System.out.print("Add more items? (yes/no): ");
    String choice = input.next().toLowerCase();
    if (choice.equals("no"))
    break;

    }
    while (index < 100);

    double totalPrice = 0;
    for (int count = 0; count < index; count++){
    totalPrice += total[count];
}

    double discount = totalPrice * 0.10;
    double vat = totalPrice * 0.075;
    double billTotal = totalPrice - discount + vat;

    System.out.println("\n===== RECEIPT =====");
    System.out.println("Customer name: " + customer);

    for (int count = 0; count < index; count++) {
    System.out.println(userItems[count] + " x" + items[count] + " @ " + price[count] + " = " + total[count]);
}

    System.out.println("-------------------------");
    System.out.printf("totalPrice: %.2f\n", totalPrice);
    System.out.printf("Discount: %.2f\n", discount);
    System.out.printf("VAT (7.5%%): %.2f\n", vat);
    System.out.printf("Bill Total: %.2f\n", billTotal);

    System.out.print("Amount Paid: ");
    double paid = input.nextDouble();

    double balance = paid - billTotal;

    System.out.printf("Balance: %.2f\n", balance);
    System.out.println("==================");
    System.out.println("    THANK YOU");

    }
}
